common/optional.hpp

```diff
@@ -15,6 +15,18 @@
 #ifndef MENDER_COMMON_OPTIONAL_HPP
 #define MENDER_COMMON_OPTIONAL_HPP
 
+#include <version>
```

Comment:
We don't use anything from `<version>`, do we? I only see our custom macro.

Reply:
You're right, it was a leftover. Removed.